ui/src/main/webapp/src/app/components/form.component.ts

```diff
@@ -24,15 +24,17 @@ export class FormComponent {
     handleError(error:any) {
         this.errorMessages = [];
         if (!error) {
-            this.errorMessages.push("Server call failed!");
-        } else if (error.parameterViolations) {
+            this.errorMessages.push("Server call failed.");
+        }
+        else if (error.parameterViolations) {
             error.parameterViolations.forEach(violation => {
                 console.log("Violation: " + JSON.stringify(violation));
                 this.errorMessages.push(violation.message);
             });
-        } else {
+        }
+        else {
```

Comment:
That's not much readable. Also, why have different standards for Java and TypeScript? Either one is more readable and the other is not. That's not context-dependent, the syntax is almost the same.